plugins/modules/cloud/scaleway/scaleway_container.py

```diff
@@ -0,0 +1,417 @@
+      - fr-par
+      - nl-ams
+      - pl-waw
+
+  name:
+    type: str
+    description:
+      - Name of the container namespace.
+    required: true
+
+  description:
+    description:
+      - Description of the container namespace.
+    type: str
+
+  min_scale:
+    description:
+      - Minimum number of replicas for the container.
+    type: int
+
+  max_scale:
+    description:
+      - Maximum number of replicas for the container.
+    type: int
+
+  environment_variables:
+    description:
+      - Environment variables of the container namespace.
+      - Injected in container at runtime.
+    type: dict
+
+  secret_environment_variables:
+    description:
+      - Secret environment variables of the container namespace.
+      - Updating thoses values will not output a C(changed) state in Ansible.
+      - Injected in container at runtime.
+    type: dict
+
+  memory_limit:
+    description:
+      - Resources define performance characteristics of your container.
+      - They are allocated to your container at runtime.
+    type: int
+
+  container_timeout:
+    description:
+      - The length of time your handler can spend processing a request before being stopped.
+    type: str
+
+  privacy:
+    description:
+      - Privacy policies define whether a container can be executed anonymously.
+      - Choose C(public) to enable anonymous execution, or C(private) to protect your container with an authentication mechanism provided by the Scaleway API.
+    type: str
+    default: public
+    choices:
+      - public
+      - private
+
+  registry_image:
+    description:
+      - The name of image used for th container.
+    type: str
+    required: true
+
+  max_concurrency:
+    description:
+      - Maximum number of connections per container.
+      - This parameter will be used to trigger autoscaling.
+    type: int
+
+  protocol:
+    description:
+      - Communication protocol of the container.
+    type: str
+    default: http1
+    choices:
+      - http1
+      - h2c
+
+  port:
+    description:
+      - Listen port used to expose the container.
+    type: int
+
+  redeploy:
+    description:
+      - Redeploy the container if update is required.
+    type: bool
+    default: false
+'''
+
+EXAMPLES = '''
+- name: Create a container
+  community.general.scaleway_container:
+    namespace_id: '{{ scw_container_namespace }}'
+    state: present
+    region: fr-par
+    name: my-awesome-container
+    registry_image: rg.fr-par.scw.cloud/funcscwtestrgy2f9zw/nginx:latest
+    environment_variables:
+      MY_VAR: my_value
+    secret_environment_variables:
+      MY_SECRET_VAR: my_secret_value
+  register: container_creation_task
+
+- name: Make sure container is deleted
+  community.general.scaleway_container:
+    namespace_id: '{{ scw_container_namespace }}'
+    state: absent
+    region: fr-par
+    name: my-awesome-container
+'''
+
+RETURN = '''
+data:
+    description: This is only present when C(state=present)
```

Comment:
```suggestion
    description: The container description.
```
That it's only there when `state=present` is already mentioned in `returned`.

Reply:
corrected on all modules